parser.py

```diff
@@ -434,54 +454,54 @@ def parse_tweets(input_filenames, username, users, html_template, archive_media_
         with open(f'{filename}.md', 'w', encoding='utf-8') as f:
             f.write(md_string)
 
-        # Write into *.html files
-        html_string = '<hr>\n'.join(html for _, html in content)
-        with open(f'{filename}.html', 'w', encoding='utf-8') as f:
-            f.write(html_template.format(html_string))
+    # Write into *.html files
+    html_string = '<hr>\n'.join(html for _, html in content)
+    with open(f'{filename}.html', 'w', encoding='utf-8') as f:
+        f.write(html_template.format(html_string))
```

Comment:
Original indentation was correct - `content` comes from the for-loop.

Reply:
Good catch! This got lost during the semi-automatic three-way-merge.

Reply:
What kind of language is this that doesn't have braces? :(

Reply:
Try this yourself:
```
>>> from __future__ import braces
  File "<stdin>", line 1
SyntaxError: not a chance
```

Reply:
I like to check occasionally. Maybe they will change their mind one day...